### DISCUSSION TITLE
„show more“ on mobile only

### DISCUSSION THREAD

**Discussion by MDRefiner** [tomik23/show-more#62] [Ideas]
Hi, your script is ranking #1 on google and i assume for a reason. Congrats on the impressive setting options.

This is a enhancement suggestion for a setting option for "breakpoint", i.e. if breakpoint has a value then showmore would only work up to the given value. And we could add a window.resize listener.

What do you think?

**Comment by tomickigrzegorz** [comment]
Hi,
You're making things a bit more complicated :)
You can check the browser width yourself or check [Window.matchMedia()](https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia) and even better check if the device is mobile - [Mobile device detection](https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent) and then run the library.

It doesn't make sense to add this to the library in my opinion. I want the library to be as small as possible and adding it inside such a solution will definitely enlarge it.

But thanks for the suggestion, maybe I'll add how to solve such a problem in the "README".

Regards
Greg
